otter/integration/tests/test_lbsh.py

```diff
@@ -599,3 +602,54 @@ def test_changing_disowned_server_is_not_converged_2(self):
                 timeout=timeout_default
             )
         ])
+
+    @inlineCallbacks
+    def test_draining(self):
```

Comment:
Do you expect this test to pass against production, given it has those strict timing requirements?  Maybe this test should have a `@skip_if(not_mimic, "Requires mimic for timing")` or something?